frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -184,6 +176,22 @@ class DocumentList extends Component {
     return this.renderDocumentList()
   }
 
+  componentWillMount() {
+    this.checkStatusAndLoad()
+  }
+
+  checkStatusAndLoad() {
```

Comment:
Super minor thing: I'd either name this `checkStatusAndFetch` or rename `fetchDocuments` to `loadDocuments`, just so that it's clear that one thing triggers the other.